src/index.js

```diff
@@ -27,6 +27,7 @@ async function bundleMDX(
     xdmOptions = (vfileCompatible, options) => options,
     esbuildOptions = options => options,
     globals = {},
+    cwd,
```

Comment:
What would happen if we just default this?

```suggestion
    cwd = path.join(process.cwd(), `__mdx_bundler_fake_dir__`),
```

And then rather than `dir` below, we'd just use `cwd` and then we could delete our own custom error logic and let esbuild just try to resolve it? Could clean things up for us a bit and as long as the error message isn't confusing I think it could be ok. What do you think?

Reply:
It works fine, only fault in the tests is that we don't get our custom errors and instead just get the esbuild errors.

Only issue here might be that esbuild doesn't report where the import error came from.